### ISSUE TITLE
Strange pixel row on top of every window

### ISSUE THREAD

**Issue by nicolas-p** [factor/factor#1620]
I have just bought a Retina Mac, which has allowed me to spot a bug that I couldn't see before.

When you start Factor, the Listener window is perfectly rendered. But every other window you open (even another Listener) has a single row of random pixels just under the title bar. The pixels in this row change colour when you move the cursor.

As soon as you resize the window, this row disappears.

I checked version 0.97: the bug was already there, so it's not because of my modifications.

A screenshot of the problem:

<img width="542" alt="capture d ecran 2016-04-27 a 20 23 11" src="https://cloud.githubusercontent.com/assets/11431580/14863448/01e40588-0cb6-11e6-8098-488e2f2b59e4.png">

**Comment by mrjbq7** [comment]
Yep a known issue that I haven't tracked down.  Resizing the window appears to "fix" it. I think has to do with 0.5 pixel GL coordinates on retina screen.

**Comment by mrjbq7** [comment]
Thanks for filing the bug so we can remember to fix it.

**Comment by nicolas-p** [comment]
The ugly pixel row disappears if I do this:

```factor
[ world? ] instances [ [ [ ceiling ] map ] change-dim ] each
```

**Comment by mrjbq7** [comment]
That would make sense, somewhere we're probably getting a ``0.5`` dimension.  That's some good information for getting this fixed.  I'm taking a 2-week course right now that's get me busy, but I'll try and make some time later.

**Comment by mrjbq7** [comment]
Fixed in 3edf8f594a3e52a70db45a6e66873d070f6f4051.
